iModelCore/GeomLibs/geom/build/inc/GeomTargets.mki

```diff
@@ -893,5 +895,4 @@ $(OUT_DIR)BSIRect$(OUT_EXT)         :$(geomSrcIntegerTYpes)BSIRect.cpp $(OTHER_D
 
 OUT_DIR = BAD_OUT_DIR_VALUE_INHERITED_FROM_SOURCES.MKI
 OUT_EXT = BAD_OUT_EXT_VALUE_INHERITED_FROM_SOURCES.MKI
-OTHER_DEPENDENCIES = BAD_OTHER_DEPENDENCIES_VALUE_INHERITED_FROM_SOURCES.MKI
-

```

Comment:
Be careful removing blank lines from makefiles, as they often have meaning. This one's probably safe, but in general I prefer not to remove a blank line at the end of a file.